### PR TITLE
todo bot cleanup December 2020

### LINKED ISSUE #162: error running parameter labels vignette interactively
An error occurs when attempting to run the "parameter labels" vignette interactively.

param_labels

Specifically, when a user attempts to run the `set_model_directory` chunk, the `setup` chunk is executed first. The `source()` statement in this chunk throws an error. The `source()`d file calls the `block()` function, which is exported from `rbabylon`, but because the `setup` chunk is run before `rbabylon` is loaded, the name cannot be resolved.

This behavior is a consequence of the [special handling](https://community.rstudio.com/t/what-defines-the-setup-chunk/27595/5) for the `setup` chunk in the RStudio IDE. Specifically, when in notebook mode, i.e., interactively running the document, the `setup` chunk will be run once before any other code (to see the difference, change the label, e.g., to `setup2`, and observe the behavior). Note that the vignette builds correctly non-interactively, because the special handling for `setup` does not apply.

We might consider either renaming the `setup` chunk, or reordering the vignette so that the `source()` call happens after loading `rbabylon`.

### LINKED ISSUE #199: consider adding a test for logs with more than one model
https://github.com/metrumresearchgroup/rbabylon/blob/9e77752da0713516c916d7d800fa225397590b21/tests/testthat/test-get-path-from-object.R#L26-L31

---

###### This issue was generated by [todo](https://todo.jasonet.co) based on a `TODO` comment in 9e77752da0713516c916d7d800fa225397590b21 when #195 was merged. cc @metrumresearchgroup.

### LINKED ISSUE #200: consider whether to delete this test now that we have done https://github.com/me...
https://github.com/metrumresearchgroup/rbabylon/blob/9e77752da0713516c916d7d800fa225397590b21/tests/testthat/test-workflow-bbi.R#L137-L141

---

###### This issue was generated by [todo](https://todo.jasonet.co) based on a `TODO` comment in 9e77752da0713516c916d7d800fa225397590b21 when #195 was merged. cc @metrumresearchgroup.

### LINKED ISSUE #215: eventually we will only exclude ABS_MOD_PATH, because that should
https://github.com/metrumresearchgroup/rbabylon/blob/004c885e6fdcf10e275d9bf5df83c44376939059/tests/testthat/test-new-model.R#L87-L92

---

###### This issue was generated by [todo](https://todo.jasonet.co) based on a `TODO` comment in 004c885e6fdcf10e275d9bf5df83c44376939059 when #213 was merged. cc @metrumresearchgroup.

### LINKED ISSUE #219: this test needs to clean up after itself by removing the files it
https://github.com/metrumresearchgroup/rbabylon/blob/f8fd75320dd52249acba2f5ed808be8b8014f8f7/tests/testthat/test-workflow-bbi.R#L51-L56

---

###### This issue was generated by [todo](https://todo.jasonet.co) based on a `TODO` comment in f8fd75320dd52249acba2f5ed808be8b8014f8f7 when #217 was merged. cc @metrumresearchgroup.

### LINKED ISSUE #220: isolate this test so it does not depend on a model created by a
https://github.com/metrumresearchgroup/rbabylon/blob/f8fd75320dd52249acba2f5ed808be8b8014f8f7/tests/testthat/test-workflow-bbi.R#L82-L86

---

###### This issue was generated by [todo](https://todo.jasonet.co) based on a `TODO` comment in f8fd75320dd52249acba2f5ed808be8b8014f8f7 when #217 was merged. cc @metrumresearchgroup.

### PR DESCRIPTION
**Pull request by seth127**
Knocking out some small housekeeping issues, mostly created by the TODO bot.

Closes #162 
Closes #199 
Closes #215
Closes #200 
Closes #219 
Closes #220

### CODE REVIEW

File: vignettes/getting-started.Rmd [hunk 1]
```diff
@@ -19,7 +19,14 @@ knitr::opts_chunk$set(
 
 This "Getting Started with rbabylon" vignette takes the user through some basic scenarios for modeling with NONMEM using `rbabylon`, introducing you to its standard workflow and functionality. 
 
-```{r results = 'hide', message=FALSE, warning=FALSE}
+```{r setup, include=FALSE}
+# NOTE: if running chunks interactively we need to load the package first
+#   because renv isolation prevents us from finding an rbabylon installation
+if (interactive()) {
+  devtools::load_all()
```
Comment (copernican) [inline vignettes/getting-started.Rmd:26]: This results in a note:

> N  checking for unstated dependencies in vignettes (1m 5s)
   '::' or ':::' import not declared from: ‘devtools’

We can [resolve](https://stackoverflow.com/a/9900160) this by adding devtools to `Suggests`.

**Review by copernican — CHANGES_REQUESTED** [review]
Looks good in general, we should make one small change to avoid a note.

**Review by copernican — APPROVED** [review]
LGTM

File: vignettes/getting-started.Rmd [hunk 2]
```diff
@@ -19,7 +19,14 @@ knitr::opts_chunk$set(
 
 This "Getting Started with rbabylon" vignette takes the user through some basic scenarios for modeling with NONMEM using `rbabylon`, introducing you to its standard workflow and functionality. 
 
-```{r results = 'hide', message=FALSE, warning=FALSE}
+```{r setup, include=FALSE}
+# NOTE: if running chunks interactively we need to load the package first
+#   because renv isolation prevents us from finding an rbabylon installation
```
Comment (dpastoor) [inline vignettes/getting-started.Rmd:24]: where did this conclusion come from?

File: tests/testthat/test-get-path-from-object.R [hunk 3]
```diff
@@ -20,10 +20,14 @@ test_that("get_yaml_path() builds the right path", {
   expect_identical(get_yaml_path(MOD1), normalizePath(YAML_TEST_FILE))
 })
 
-
-# TODO: consider adding a test for logs with more than one model
 test_that("get_model_path() works with bbi_*_log_df", {
-  expect_identical(get_model_path(run_log(MODEL_DIR)), normalizePath(CTL_TEST_FILE))
+  create_all_models()
+  on.exit(cleanup())
```
Comment (dpastoor) [inline tests/testthat/test-get-path-from-object.R:25]: use defer https://testthat.r-lib.org/articles/test-fixtures.html#withrdefer

**Review by dpastoor — COMMENTED** [review]
I'd like to understand this whole load_all() stuff

**Pull request merged**